addons/mail/views/mail_template_views.xml

```diff
@@ -34,16 +34,20 @@
                         </div>
                         <div class="oe_title">
                             <label for="name"/>
-                            <h1><field name="name" required="1" placeholder="e.g. Calendar: Reminder"/></h1>
+                            <h1><field name="name" required="1" placeholder="e.g. Calendar: First Contact"/></h1>
                             <group>
-                                <field name="model_id" placeholder="e.g. Users" required="1" options="{'no_create': True}"/>
+                                <field name="model_id" placeholder="e.g. Contact" required="1" options="{'no_create': True}"/>
                                 <field name="model" invisible="1"/>
                             </group>
                         </div>
                         <notebook>
                             <page string="Content" name="content">
-                                <label for="subject"/>
-                                <h2 style="display: inline-block;"><field name="subject" placeholder="Subject (placeholders may be used here)"/></h2>
+                                <div class="d-block d-sm-flex">
```

Comment:
That's strange as we avoid doing that kind of custom display in a standard form view. As normally labels take only a minimal space, what are you trying to solve exactly ?

Reply:
I'm trying to make the form more readable on small screen: With those classes, the label will be above the field on small screen size. The other fields of the form are also positioned that way :)

Reply:
If we have troubles with mobile, shouldn't it be managed at form view level ? I mean, we should not add tweaks in every form view but ensure form view are correctly displayed on mobile... Did that come from a request or a bug report ?